src/System.Private.ServiceModel/tools/IISHostedWcfService/App_code/testhosts/CustomTextEncoderBufferedServiceHost.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Reviewers: this file is effectively the same as what was in the old self-hosted server's old TestResources folder, refactored to follow the pattern of the other Resource ports to the new test infrastructure.  There are no changes to the way the Binding is created or the Address.  The same is true for the Streamed version of this service.